### PR TITLE
[#172050222] add prescription metadata into message model

### PR DESCRIPTION
**Pull request by gunzip**

### CODE REVIEW

File: openapi/definitions.yaml [hunk 1]
```diff
@@ -106,6 +108,29 @@ NewMessage:
       $ref: "#/FiscalCode"
   required:
     - content
+PrescriptionData:
+  type: object
+  description: |-
+    Metadata needed to process medical prescriptions.
+  properties:
+    nre:
+      $ref: "#/PrescriptionNre"
```
Comment (cloudify) [inline openapi/definitions.yaml:117]: I believe `Nre` is an acronym, it should be all caps

File: openapi/definitions.yaml [hunk 2]
```diff
@@ -106,6 +108,29 @@ NewMessage:
...
+    iup:
+      $ref: "#/PrescriptionIup"
```
Comment (cloudify) [inline openapi/definitions.yaml:119]: I believe `Iup` is an acronym, it should be all caps

File: openapi/definitions.yaml [hunk 3]
```diff
@@ -106,6 +108,29 @@ NewMessage:
...
+    doctor_fiscal_code:
```
Comment (cloudify) [inline openapi/definitions.yaml:120]: in English, it's called `md` (Medicinae Doctor), so it should be `md_fiscal_code`, or perhaps if we want to be a little more generic we can use `prescriber_fiscal_code`.

Comment (gunzip) [reply openapi/definitions.yaml:120]: I'm ok with this but since these specs are already under review let's double check with @matteodesanti

**Comment by gunzip** [comment]
closing in favor of #64

**Pull request closed without merging**